packages/sdk/src/utils.ts

```diff
@@ -19,7 +19,7 @@ const parseEntry = (str: string, devVerOrUrl?: string): RemoteEntryInfo => {
     getProcessEnv()['NODE_ENV'] === 'development' && devVerOrUrl;
   const defaultVersion = '*';
   const isEntry = (s: string) =>
-    s.startsWith('http') || s.endsWith(MANIFEST_EXT);
+    s.startsWith('http') || s.includes(MANIFEST_EXT);
```

Comment:
if these are always valid urls, we could use new URL() for parse? 
However i dont think we will run into any issues with includes

Reply:
the url maybe is a version like "0.0.2" , so it can not use new URL to parse